### PR TITLE
Fix Micrometer configuration documentation issues

### PR DESCRIPTION
**Pull request by loicmathieu**
- `<p>` is not allowed in asciidoc
- moving one doc to asciidoc for better display of code

### CODE REVIEW

**Review by ebullient — APPROVED** [review]

File: extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/config/MicroprofileMetricsConfig.java [hunk 1]
```diff
@@ -10,23 +10,28 @@
  */
 @ConfigGroup
 public class MicroprofileMetricsConfig implements MicrometerConfig.CapabilityEnabled {
+    // @formatter:off
     /**
      * Microprofile Metrics support.
```
Comment (gsmet) [inline extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/config/MicroprofileMetricsConfig.java:15]:
```suggestion
     * Eclipse MicroProfile Metrics support.
```
While we're at it (we need Eclipse the first time we mention it).

File: extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/config/MicroprofileMetricsConfig.java [hunk 2]
```diff
@@ -10,23 +10,28 @@
...
-     * <p>
+     *
      * Support for Microprofile metrics will be enabled if micrometer
```
Comment (gsmet) [inline extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/config/MicroprofileMetricsConfig.java:17]:
```suggestion
     * Support for MicroProfile Metrics will be enabled if Micrometer
```

File: extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/config/MicroprofileMetricsConfig.java [hunk 3]
```diff
@@ -10,23 +10,28 @@
...
-     * <p>
+     * [source,xml]
+     * ----
+     * <dependency>
+     *   <groupId>org.eclipse.microprofile.metrics</groupId>
+     *   <artifactId>microprofile-metrics-api</artifactId>
+     * </dependency>
+     * ----
+     *
      * The micrometer extension currently provides a compatibility layer that supports the MP Metrics API,
```
Comment (gsmet) [inline extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/config/MicroprofileMetricsConfig.java:28]:
```suggestion
     * The Micrometer extension currently provides a compatibility layer that supports the MP Metrics API,
```

**Review by gsmet — CHANGES_REQUESTED** [review]
While we're at it :)

**Comment by loicmathieu** [comment]
OK, so I applied the suggestion from @gsmet and capitalize all the `Micrometer` on the various config item docs.

**Review by ebullient — APPROVED** [review]
Thank you for doing this. I could swear I've gone through to do this, but obviously not thoroughly enough. ;)

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Thanks!

**Pull request merged**